src/channel.ts

```diff
@@ -74,6 +78,8 @@ export function getSupportedMark(channel: Channel): SupportedMark {
   switch (channel) {
     case X:
     case Y:
+    case X2:
```

Comment:
`X2`, `Y2`  need their own group.   `point`, `tick`, `circle`, `square`, `line`, `text` should not support x2, y2.